### PR TITLE
[misc] fix express deprecations

### PR DESCRIPTION
**Pull request by das7pad**
### Description

For https://github.com/overleaf/issues/issues/2965

I paired with @warncke on this.

#### Related Issues / PRs

https://github.com/overleaf/issues/issues/2965

#### Potential Impact

High, but all usage should be covered by unit/acceptance tests.

### CODE REVIEW

File: test/unit/js/HttpControllerTests.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ describe('HttpController', function() {
     })
     this.res = {
       send: sinon.stub(),
```
Comment (mans0954) [inline test/unit/js/HttpControllerTests.js:38]: Do you still need the `send` stub?

Comment (das7pad) [reply test/unit/js/HttpControllerTests.js:38]: Yes.

<details>

<pre>
  1) HttpController
       getRawDoc
         "before each" hook for "should get the document without the version":
     TypeError: res.send is not a function
      at /app/app/js/HttpController.js:62:20
      at callCallback (node_modules/sinon/lib/sinon/behavior.js:101:25)
      at Object.invoke (node_modules/sinon/lib/sinon/behavior.js:146:27)
      at Object.functionStub (node_modules/sinon/lib/sinon/stub.js:39:43)
      at Function.invoke (node_modules/sinon/lib/sinon/proxy-invoke.js:47:47)
      at Object.functionStub [as getDocLines] (node_modules/sinon/lib/sinon/proxy.js:215:26)
      at Object.getRawDoc (app/js/HttpController.js:54:23)
      at Context.<anonymous> (test/unit/js/HttpControllerTests.js:139:34)

  2) HttpController
       updateDoc
         when the doc body is too large
           "before each" hook for "should return a 413 (too large) response":
     TypeError: res.status(...).send is not a function
      at Object.updateDoc (app/js/HttpController.js:143:23)
      at Context.<anonymous> (test/unit/js/HttpControllerTests.js:419:36)
</pre>
</details>

**Comment by das7pad** [comment]
Zenhub: putting this in Help for a deploy on Monday.

**Comment by das7pad** [comment]
Force pushed to fix a conflict.

**Pull request merged**